.github/workflows/test-queries.yml

```diff
@@ -4,9 +4,15 @@ on:
   push:
     branches:
       - "master"
+    paths:
+      - "lockfile.json"
+      - "queries/**"
```

Comment:
We already have a cache for regenerating the parsers, changes in the build/install logic should also trigger building and tests.

It's very tricky to discover bugs when a important file of the cache was forgotten. Not sure if the paths could be inverted to only apply to your documentation usecase

Reply:
Same holds for changes to parser.lua which add a new parser. Would try to use the same pattern as for the parser build cache plus queries